yolact.py

```diff
@@ -470,6 +131,16 @@ def __init__(self):
         self.detect = Detect(cfg.num_classes, bkg_label=0, top_k=cfg.nms_top_k,
             conf_thresh=cfg.nms_conf_thresh, nms_thresh=cfg.nms_thresh)
 
+
+        # set default backbone weights
+        self.init_weights(backbone_path='weights/' + cfg.backbone.path)
+
+        # GPU
+        #TODO try half: net = net.half()
+        self.cuda()
```

Comment:
We should probably either pass in the device as an argument or let the user handle this externally. Just calling `.cuda()` has pytorch pick a GPU, and it restricts future CPU or TPU support.

Reply:
> and to add arbitrary device support (basically change all the .cuda() and fix all the tensor initializations). 

I'll need some guidance on this. Do you mean just add `Yolact(device="gpu/cpu/tpu")` and handle these accordingly?

Reply:
I like this and prepared support for the feature is in 236352d , 
But there's a lot of `.cuda()` calls in the code so far, even in external DCNv2. I'd prefer to finish this feature in a separate PR later. 
Btw, the alternative to `.cuda()` is `.to(device="gpu")` ..if so, that's what we'd use.